--- origsrc/rest-0.7.10/rest/Makefile.am	2011-03-16 12:25:11.000000000 -0500
+++ src/rest-0.7.10/rest/Makefile.am	2011-05-18 02:13:48.848974500 -0500
@@ -40,6 +40,7 @@ librest_@API_VERSION@_la_CFLAGS = $(GLIB
 librest_@API_VERSION@_la_LIBADD = $(GLIB_LIBS) $(GTHREAD_LIBS) \
                     $(SOUP_LIBS) $(SOUP_GNOME_LIBS) $(XML_LIBS) \
 		    $(GCOV_LDFLAGS)
+librest_@API_VERSION@_la_LDFLAGS = -no-undefined
 librest_@API_VERSION@_la_SOURCES = $(lib_sources) $(lib_headers)
 librest_@API_VERSION@_la_HEADERS = $(lib_headers)
 librest_@API_VERSION@_ladir = $(includedir)/rest-@API_VERSION@/rest
--- origsrc/rest-0.7.10/rest-extras/Makefile.am	2011-03-31 15:28:00.000000000 -0500
+++ src/rest-0.7.10/rest-extras/Makefile.am	2011-05-18 02:14:06.712996300 -0500
@@ -26,6 +26,7 @@ librest_extras_@API_VERSION@_la_LIBADD =
                     $(SOUP_LIBS) $(SOUP_GNOME_LIBS) $(XML_LIBS) \
 		    $(GCOV_LDFLAGS) \
                     $(top_builddir)/rest/librest-@API_VERSION@.la
+librest_extras_@API_VERSION@_la_LDFLAGS = -no-undefined
 librest_extras_@API_VERSION@_la_SOURCES = $(lib_sources) $(lib_headers)
 librest_extras_@API_VERSION@_la_HEADERS = $(lib_headers)
 librest_extras_@API_VERSION@_ladir = $(includedir)/rest-@API_VERSION@/rest-extras
